tests/__snapshots__/files/prism-core.js.shot

```diff
@@ -464,10 +464,12 @@ var Prism = (function (_self) {
 		 * 3. All hooks of {@link Prism.highlightElement} for each element.
 		 *
 		 * @memberof Prism
-		 * @param {ParentNode} container The root element, whose descendants that have a \`.language-xxxx\` class will be highlighted.
+		 * @param {ParentNode} container
+		 *   The root element, whose descendants that have a \`.language-xxxx\` class will be highlighted.
```

Comment:
Cases similar to this one can be seen in a few other places as well. They are caused by [this function allowing for 10 extra chars](https://github.com/hosseinmd/prettier-plugin-jsdoc/blob/54bdbc54614d2919f426a926b0612de47d5e5fdd/src/descriptionFormatter.ts#L219) in the last line so the description doesn't get broken into 2 lines as it should be. 

`stringify` checks and sees that the tag will go over `printWidth` if it appends the description to the rest of the tag as is. Since the description is too long to fit, `stringify` will add a line break before the description. This is why this is broken in such a weird way.

The fix is simple. Don't allow the extra 10 chars.